Makefile

```diff
@@ -316,7 +328,7 @@ set-test-image-vars:
 .PHONY: set-test-operator-config
 set-test-operator-config:
 	# we always install the OpenShift Route controller for e2e tests (in the prepare-e2e step)
-	sed -i 's/openshiftRoute: false/openshiftRoute: true/' config/manager/controller_manager_config.yaml
+	sed -i 's/openshiftRoute: false/openshiftRoute: true/' config/overlays/community/config/controller_manager_config.yaml
```

Comment:
We need to move the route test to openshift suite. I would prefer to have a separate PR for that.

Reply:
Created issue for this: https://github.com/os-observability/tempo-operator/issues/350